content/ko/docs/contribute/localization_ko.md

```diff
@@ -328,16 +361,20 @@ skew | 차이(skew) |
 spec | 명세, 스펙, 사양 |
 specification | 명세 |
 Stateful Set | 스테이트풀 셋 |
```

Comment:
Maybe remove “Stateful Set” in favor of “StatefulSet”?

Reply:
I agree with you.